infra.lvn_tools/extend_boot/README.md

```diff
@@ -0,0 +1,40 @@
+# ansible-bash-extend-boot
+Ansible role that runs the extend boot partition script. The role contains the shell scripts to extend the boot partition, as well as the script wrapping it to run as part of the pre-mount step during the boot process. Finally, there is a copy of the `sfdisk` binary with version 2.38.1 to ensure the extend script will work regardless of the `util-linux` package installed in the target host. 
```

Comment:
I think the first sentence needs rephrasing

Reply:
Done